packages/dataviews/src/dataforms-layouts/panel/index.tsx

```diff
@@ -44,12 +44,10 @@ function DropdownHeader( {
 				<Spacer />
 				{ onClose && (
 					<Button
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
-						className="dataforms-layouts-panel__dropdown-header-action"
 						label={ __( 'Close' ) }
 						icon={ closeSmall }
 						onClick={ onClose }
+						size="small"
```

Comment:
Enable the "Quick Edit in DataViews" experiment. Go to the list of Pages in Site Editor and enable the Table view. Toggle the details panel to the right, and click on a field to show the edit popover.

No visual changes:

<img src="https://github.com/user-attachments/assets/70a88551-da7d-4335-8605-fc578af5b69c" alt="Quick edit popover" width="722">